src/main.ts

```diff
@@ -27,7 +27,15 @@ async function run(): Promise<void> {
     }
 
     stateHelper.setCleanup(inputs.cleanup);
-    await exec.exec(`${daggerBin} ${inputs.args} --log-format plain`);
+
+    if (inputs.args) {
+      inputs.cmds.unshift(inputs.args);
+      core.warning(`'args' is being deprecated, please use 'cmds'`);
+    }
+
+    for (const cmd of inputs.cmds) {
+      await exec.exec(`${daggerBin} ${cmd} --log-format plain`);
```

Comment:
Would be better to wrap exec output in foldable groups: https://github.com/actions/toolkit/blob/main/packages/core/README.md#logging

```suggestion
	  await core.group(cmd, async () => {
	    await exec.exec(`${daggerBin} ${cmd} --log-format plain`);
      });
```